meta/devices/friendlyarm_nanopi-r1.yaml

```diff
@@ -0,0 +1,43 @@
+vendor: FriendlyARM
+model: NanoPi R1
+bootloader: unknown
+cpu:
+  model: Allwinner H3
+  cores: 4
+  mhz: 1200
+memory_mb:
+  - 512
+  - 1024
+storage:
+  - type: microSD
+device_type: WiFi Router
+usb:
+  - type: A
+    ports: 2
+ethernet:
+  - speed_mbit: 100
+    ports: 1
+    label: lan
+power_supply:
+  type: DC
+  volt: 5
+  ampere: 2
```

Comment:
Should that be the minimum requirement of the router or the value printed on the PSU adapter (i.e. more than necessary?)

Reply:
Context: E.g. outdoor devices with PoE passthrough have PSUs that provide enough power for two devices.

Reply:
I think value printed on PSU adapter. It's hard to tell the real minimum, not?

Reply:
>  It's hard to tell the real minimum, not?

Yes. But for my "Context" example above the value from PSU would not be so much help. Of course, this is all nitpicking. But again, this is essentially the Wiki vs. firmware-related parameter discussion.

Reply:
> Should that be the minimum requirement of the router

How would you know that? Technical specs found on OEM websites tend to be influenced by marketing guys, resulting in some OEMs providing no info at all, or even silly "110-240 VAC" while the device has a AC/DC power adapter.
We are happy about every information we get, and we are happy about every OEM that doesn't contradict himself on his website vs. owners manual etc.

Enter what is available, i.e. whatever is printed on the device itself, on the PSU, or in the owners manual.

Reply:
And that's why I wouldn't provide that info in the repo.

Of course, this folds back to the question whether we will actually have the whole TOH in master; if that's the goal eventually, I would not _delete_ useful information.

BTW: how would we deal with outdated metadata in stable releases? Assuming somebody would parse it, would we backport meta data changes??

Reply:
What is written on the device matters, because PSUs are often bulk-ware shared among different products and not necessarily the best fit in terms of energy efficiency. Please don't use the PSU values if there is no other reference. At least in the EU vendors are required to print the needed voltage/amps on the device or in the manual for these kind. The included power supply must have a specific efficiency with the device, but it can have a higher max amp value.